db/app/app.test.js

```diff
@@ -45,6 +45,7 @@ describe("get /api/articles", () => {
         expect(article).toHaveProperty("created_at");
         expect(article).toHaveProperty("votes");
         expect(article).toHaveProperty("article_img_url");
+        expect(article).toHaveProperty("comment_count")
```

Comment:
Should test the length of the response array and that all of the returned articles have the correct properties

Reply:
test that results come back in order. (think Jest-Sorted)

Reply:
we should be testing the type of the value on each key too